ha/monitor/k8s/parser.py

```diff
@@ -23,20 +23,30 @@
 from ha.monitor.k8s.const import AlertStates
 from ha.monitor.k8s.const import EventStates
 
+from ha.core.config.config_manager import ConfigManager
+from ha.const import _DELIM, HA_GLOBAL_INDEX
+from cortx.utils.conf_store import Conf
 from cortx.utils.log import Log
 from ha.fault_tolerance.const import HEALTH_EVENT_SOURCES, NOT_DEFINED
 from cortx.utils.event_framework.health import HealthAttr, HealthEvent
 
 
 class ObjectParser:
     def __init__(self):
+        # Load config
+        ConfigManager.init(None)
+        # TODO: Should be fetched from confstore
+        cluster_id = Conf.get(HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}cluster_id")
+
+        # Default value will be ''(empty string) instead of None, as None will decode as
+        # 'null' in json.dumps and may failed ast.literal_eval
         self.payload = {HealthAttr.SOURCE.value: HEALTH_EVENT_SOURCES.MONITOR.value,
-                    HealthAttr.CLUSTER_ID.value: None,
+                    HealthAttr.CLUSTER_ID.value: cluster_id,
                     HealthAttr.SITE_ID.value: NOT_DEFINED,
                     HealthAttr.RACK_ID.value: NOT_DEFINED,
                     HealthAttr.STORAGESET_ID.value: NOT_DEFINED,
-                    HealthAttr.NODE_ID.value: None, HealthAttr.RESOURCE_TYPE.value: None,
-                    HealthAttr.RESOURCE_ID.value: None, HealthAttr.RESOURCE_STATUS.value: None,
+                    HealthAttr.NODE_ID.value: '', HealthAttr.RESOURCE_TYPE.value: '',
+                    HealthAttr.RESOURCE_ID.value: '', HealthAttr.RESOURCE_STATUS.value: '',
```

Comment:
Everyone should be aware of this change from None to empty string. While consuming and validating these health attributes, I had seen check on value is None or not None, like `if value is not None:`. Instead of validating against None, the validation should be against its has value or not. Like `if not value:`

Reply:
Why do we have None to empty string change? @gauri-bhosale

Reply:
Added in comment, reason of change in None to empty string, as we are using KvPayload for event creation from utils, which supports empty string by default.

Reply:
Checked conditions for None checks in existing code, but not found particular event attributes.